src/Routes/DynamicRoute/DynamicRoute.tsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-console */
```

Comment:
Any reason why, you have no console commands in the file -- was this done during development to avoid the error in your IDE?

I know we have had disagreements about the state of the lint... I can shift it to warning if it's a workflow issue for you. Just curious why you added it, I assume just the "error" in the IDE that is annoying?

I'd just prefer us not to have a system that allows debug logs to make it into the master branch, but I don't want to interrupt workflows with lints for things that don't really hurt.

Reply:
Yeah, good point! I'll remove that. Naaah, it's fine to keep this rule. I just gotta get used to it.